src/client/models/client/HttpClient.ts

```diff
@@ -25,6 +26,21 @@ export abstract class HttpClient {
         return `${scheme}://${this.clientOptions.requestHost}`;
     }
 
+    public getHttpProxy(): AxiosProxyConfig | false {
```

Comment:
This class is an abstract class, which should be used only as a contract for http clients that plan to implement it's functionality. The main intention of this abstract class is to serve as a base contract on how to implement an http client, and one can be implemented by extending it.

This serves the purpose of being able to use any http client for Postmark Node js in future, or by users of the library.

In this PR, it seems to be relying on axios, which should not be the case. For extending axios client, this should be [done here](https://github.com/ActiveCampaign/postmark.js/blob/main/src/client/HttpClient.ts).